include/mbedtls/asn1write.h

```diff
@@ -29,11 +29,11 @@
 #define MBEDTLS_ASN1_CHK_ADD(g, f)                      \
     do                                                  \
     {                                                   \
-        if( ( ret = (f) ) < 0 )                         \
-            return( ret );                              \
+        if ((ret = (f)) < 0)                         \
```

Comment:
Minor: it would be nice if uncrustify could keep the end-of-line backslashes aligned.